backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Docs_Inout_Details_Footer.sql

```diff
@@ -38,11 +39,15 @@ FROM (
                 'docnote'                                                                                   AS tag,
                 3                                                                                           AS pozition,
                 COALESCE(inc_trl.name, inc.name)                                                            AS Incoterms,
-                io.incotermlocation
+                io.incotermlocation,
+                prt.additionaltext
          FROM m_inout io
                   LEFT JOIN c_doctype dt ON io.c_doctype_id = dt.c_doctype_id AND dt.isActive = 'Y'
                   LEFT OUTER JOIN C_Incoterms inc ON io.c_incoterms_id = inc.c_incoterms_id
                   LEFT OUTER JOIN C_Incoterms_trl inc_trl ON inc.c_incoterms_id = inc_trl.c_incoterms_id AND inc_trl.ad_language = p_Language
+                  LEFT OUTER JOIN C_BPartner_DocType pdt ON pdt.c_doctype_id = io.C_DocType_ID AND pdt.c_bpartner_id = io.c_bpartner_id
+                  LEFT OUTER JOIN C_BPartner_Report_Text prt ON pdt.c_bpartner_report_text_id = prt.c_bpartner_report_text_id
```

Comment:
pls extract this in a function that retrieve the text directly and use it all over